tests/testthat/test-deprecated.R

```diff
@@ -1,105 +1,106 @@
 ## Test deprecated opencage_forward ##
 
 test_that("opencage_forward/opencage_reverse return what they should.", {
-  withr::local_envvar(c("OPENCAGE_KEY" = key_200))
   lifecycle::expect_deprecated(
-    results <- opencage_forward(placename = "Sarzeau")
+    results <- opencage_forward(placename = "Sarzeau", key = key_200)
   )
-  expect_is(results, "list")
-  expect_is(results[["results"]], "tbl_df")
-  expect_is(results[["total_results"]], "integer")
-  expect_is(results[["time_stamp"]], "POSIXct")
+  expect_type(results, "list")
+  expect_s3_class(results[["results"]], "tbl_df")
+  expect_type(results[["total_results"]], "integer")
+  expect_s3_class(results[["time_stamp"]], "POSIXct")
   expect_equal(length(results), 4)
 
   lifecycle::expect_deprecated(
-    results <- opencage_reverse(longitude = 0, latitude = 0)
+    results <- opencage_reverse(longitude = 0, latitude = 0, key = key_200)
   )
-  expect_is(results, "list")
-  expect_is(results[["results"]], "tbl_df")
-  expect_is(results[["total_results"]], "integer")
-  expect_is(results[["time_stamp"]], "POSIXct")
+  expect_type(results, "list")
+  expect_s3_class(results[["results"]], "tbl_df")
+  expect_type(results[["total_results"]], "integer")
+  expect_s3_class(results[["time_stamp"]], "POSIXct")
   expect_equal(length(results), 4)
 })
 
 test_that("opencage_forward/opencage_reverse return what they should
           with several parameters.", {
   withr::local_envvar(c("OPENCAGE_KEY" = key_200))
+  withr::local_options(lifecycle_verbosity = "quiet")
 
-  lifecycle::expect_deprecated(
-    results <- opencage_forward(
-      placename = "Paris",
-      limit = 2,
-      min_confidence = 5,
-      language = "fr",
-      no_annotations = TRUE)
+  results <- opencage_forward(
+    placename = "Paris",
+    limit = 2,
+    min_confidence = 5,
+    language = "fr",
+    no_annotations = TRUE
   )
-  expect_is(results, "list")
-  expect_is(results[["results"]], "tbl_df")
-  expect_is(results[["total_results"]], "integer")
-  expect_is(results[["time_stamp"]], "POSIXct")
+
+  expect_type(results, "list")
+  expect_s3_class(results[["results"]], "tbl_df")
+  expect_type(results[["total_results"]], "integer")
+  expect_s3_class(results[["time_stamp"]], "POSIXct")
   expect_equal(length(results), 4)
   expect_equal(sum(grepl(
     "annotation",
     names(results[["results"]])
   )), 0)
   expect_true(dplyr::between(nrow(results[["results"]]), 1, 2))
 
-  lifecycle::expect_deprecated(
```

Comment:
is there now any expectation around the deprecation in opencage tests?

Reply:
Yes, there are still `lifecycle::expect_deprecated` calls in lines 4 and 13. I hid the explanation in this commit message [`1a0bab0` (#141)](https://github.com/ropensci/opencage/pull/141/commits/1a0bab0f7fa30584aa51ffb800ad240c07841036). 😄 

(Hope I understood your question correctly.)

Reply:
aaah ok, yes, thanks!